### DISCUSSION TITLE
Release notes (discussion)

### DISCUSSION THREAD

**Discussion by lemire** [ada-url/ada#206] [General]
We are happy to announce the first release of ada (v1.0.0) a fast WHATWG-compliant URL parser written in modern C++.

- It has been adopted by Node.js and will be part of future releases.
- The parser is fast. In some cases, it is almost twice as fast as the C++ URL parser present in recent versions of Node.js.
- The parser is correct: it passes the full [WHATWG](https://url.spec.whatwg.org/#url-parsing)  and Node.js test suites.
- Its modern and safe interface avoids bad surprises. 
- It supports both exception and exception-free usage.
- [We provide a complete and accurate documentation](https://ada-url.github.io/ada/).
- We run complet continuous-integration tests on varied systems supporting C++17: big- and little-endian, windows, macOS, Linux, etc.
- It is available both as a CMake library and as a single-header package (simply drop ada.h and ada.cpp in your project).

This release would not have been possible without the contributions of [Robert Nagy](https://github.com/miguelteixeiraa), [Miguel Teixeira](https://github.com/miguelteixeiraa) and [Anna Henningsen](https://github.com/addaleax).

**Comment by anonrig** [comment]
Let's add that `Ada` replaces the existing URL parser of Node.js
